Casks/k/kindle-create.rb

```diff
@@ -0,0 +1,28 @@
+cask "kindle-create" do
+  version "1.93.0.0"
+  sha256 :no_check
+
+  url "https://d2bzeorukaqrvt.cloudfront.net/KindleCreateInstaller.pkg",
+      verified: "https://d2bzeorukaqrvt.cloudfront.net/"
+  name "Kindle Create"
+  desc "Creating beautiful books has never been easier"
+  homepage "https://kdp.amazon.com/en_US/help/topic/GUGQ4WDZ92F733GC"
+
+  livecheck do
+    skip "No version information available"
+  end
+
+  auto_updates true
+
+  depends_on macos: ">= :catalina"
+
+  pkg "KindleCreateInstaller.pkg"
+
+  uninstall pkgutil:  "com.amazon.kc",
+
+  zap trash: [
+    "~/Library/Saved Application State/com.amazon.kc.savedState",
+    "/private/var/folders/rt/f0npbks941303yj_g47205f80000gn/C/com.amazon.kc",
+  ]
+
```

Comment:
```suggestion

  caveats do
    requires_rosetta
  end
```

It does not seem to have a release targeting Apple Silicon devices.